docs/user_guide/configuring.rst

```diff
@@ -179,6 +179,36 @@ By default, this theme's sidebar has these two elements in it. If you'd like to
 override this behavior and control the sidebar on a per-page basis, use the
 `Sphinx html-sidebars configuration value <https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=html_sidebars#confval-html_sidebars>`_.
 
+
+Configure the navigation depth and collapsing of the sidebar
+============================================================
+
+By default, this theme enables to expand/collapse subsections in the left
+sidebar navigation (without actually navigating to the page itself), and this
+up to 4 levels deep.
+
+When having a site with many files and/or many levels, this can cause a long
+build time and larger HTML file sizes. Therefore, it is possible to turn off
+the expandable navigation by setting the `collapse_navigation` config option
+to True:
```

Comment:
It feels a bit strange to say you can turn *off* the expandable navigation by setting an option to True (and in general "collapse" doesn't necessarily indicate that the actual HTML is *trimmed*), but "collapse_navigation" is consistent with RTD and consistent with sphinx' own terminology of `collapse=True/False`